src/sestest/VerifyCommand.cs

```diff
@@ -63,13 +63,13 @@ public VerifyCommand(ILogger logger) : base(logger)
 
             using (var client = new HttpClient())
             {
-                var request = new SoftwareEntitlementRequest
+                var requestBody = new SoftwareEntitlementRequestBody
```

Comment:
I'm not convinced by the rename adding the `Body` suffix, but having trouble articulating why.

Reply:
I changed it because I was getting confused between the various "request" objects.

There's the HTTP request, of which the body is just one part, and notably doesn't contain API version which I expect to be able to find in the `Request`. The rename makes it clearer to me that it's _just_ the HTTP request body, nothing else.

There's also the concept of a request at a logical level, in the `SoftwareEntitlements` layer, completely independent of transport technology. Here the rename reminds me that we're talking about something HTTP-ish.

And also we have a couple of `Response` classes (now) in the `Server` project. If we stick with the `SoftwareEntitlementRequest` naming, they appear to be the complement of each other, but actually they're completely different things.

All that said, I'm not wedded to the idea of the `Body` extension, so any other suggestions welcome.